readthedocs/oauth/models.py

```diff
@@ -174,6 +213,20 @@ class RemoteRepository(TimeStampedModel):
         _("VCS provider"), choices=VCS_PROVIDER_CHOICES, max_length=32
     )
 
+    github_app_installation = models.ForeignKey(
+        GitHubAppInstallation,
+        verbose_name=_("GitHub App Installation"),
+        related_name="repositories",
+        null=True,
+        blank=True,
+        # Delete the repository if the installation is deleted?
+        # or keep the repository and just remove the installation?
+        # I think we should keep the repository, but only if it's linked to a project,
+        # since a user could re-install the app, they shouldn't need to
+        # manually link each project to the repository again.
```

Comment:
If the user removes the installation, what would happen with remote repository permissions? Would users still be able to access the documentation?

Reply:
Currently yeah, but code isn't finished, there are a couple of decisions to take on how the app should behave when removing repos/installations.